index.ts

```diff
@@ -12,14 +12,21 @@ const {
   SCOREBOT_LEADERBOARD_URL: leaderboardUrl = '',
   SCOREBOT_PORT: port = '80',
   SCOREBOT_USE_SSL: useHttps = '0',
+  MOCK_SLACK_PORT: mockApiPort = '5010',
+  SLACK_API_TYPE: slackApiType,
 } = process.env;
 
 const protocol = useHttps !== '1' ? 'http://' : 'https://';
 const frontendUrl = protocol + leaderboardUrl;
 const server = express();
-// @ts-ignore
-setSlackClient(new slackClient.WebClient(accessToken));
 
+if (slackApiType === 'mock') {
+  // @ts-ignore
```

Comment:
I have a feeling these errors might be caused by us using the legacy client, changing to the current one might make them go away.
Otherwise, sure, a wrapper will make sense.